iTechArt.Common/Helpers/Interfaces/ILogger.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace iTechArt.Common.Helpers.Interfaces
+{
+    public interface ILogger
+    {
+        void Log(object objectToLog);
```

Comment:
I don't really like the approach with passing object. Almost always you are going to pass string or exceptions to logger. NLogger has appropriate methods. It allows us to avoid boxing/unboxing.